src/signal.cpp

```diff
@@ -189,11 +189,9 @@ static SmallSet<DiagDirection, SIG_GLOB_SIZE> _globset("_globset"); ///< set of
 
 
 /** Check whether there is a train on rail, not in a depot */
-static Vehicle *TrainOnTileEnum(Vehicle *v, void *)
+static bool TrainOnTileEnum(const Vehicle *v)
```

Comment:
Maybe rename to `IsTrainNotInDepot` or something similarly descriptive? The same function name is used in train_cmd.cpp as well.